### DISCUSSION TITLE
Add feature of converting overlap stackup to layered stackup and export project from 3dlayout to icepak for thermal simulationing

### DISCUSSION THREAD

**Discussion by louseph** [ansys/pyaedt#2605] [Ideas]
Icepak doesnt work with overlapping stackups.

Please write a feature to convert 3dlayout design with overlapping stackup to layered stackup and export to icepak design

**Comment by Samuelopez-ansys** [comment]
@louseph I think converting a layout from overlapping to layered is not simple. Please @ring630 , @svandenb-dev , do you have any idea? I think this should be performed in EDB.

**Comment by hui-zhou-a** [comment]
@louseph the layout is in overlapping mode for a reason. Some objects may not fit in laminate mode. How to deal with these objects?
